InstallGuide.md

```diff
@@ -0,0 +1,49 @@
+Canal provides two things
+
+- Networking between hosts in your cluster
+- Enforcement of network policy
+
+# Networking
+Networking can be handled either by `flannel` or by `calico`. Regardless of which is used, Calico needs to be involved when containers are started. See the [Policy](#policy) section below. 
```

Comment:
Perhaps this section should briefly explain when you'd use one or the other?

Alternatively, are we sure the we should be talking about using Calico as a networking option at this point?  Feels like until we've done the technical bits to make a canal/node which supports a BGP back-end, canal could be thought of as "flannel for networking, calico for policy".  We could tweak this when the integration is a bit tighter.